apps/nextra/utils/generated/localeMap.ts

```diff
@@ -4,22 +4,28 @@
 
 export const localeMap = {
   "/build/apis/aptos-labs-developer-portal": {
-    "en": true
```

Comment:
Not sure how the changes in this file are introduced... maybe by running one of the `pnpm` commands?